tests/test_text/test_postag.py

```diff
@@ -229,7 +230,8 @@ def test_frequency_mode(self):
         # Assert that ticks are set properly
         assert ticks_ax == sorted_tags
 
-        self.assert_images_similar(ax=ax, tol=0.5)
+        # Fails on Miniconda/Appveyor with images not close (RMS 5.302)
+        self.assert_images_similar(ax=ax, tol=5.5)
```

Comment:
I understand that updating the tolerance globally was required for both Travis and AppVeyor because Travis tests miniconda; however if you'd like to keep the more specific python-only test, we could use the is_windows_or_conda functionality that we built in for this reason:

```python
from tests.base import IS_WINDOWS_OR_CONDA

tol = 5.5 if IS_WINDOWS_OR_CONDA else 0.25
```

But it's good to note that that the tight layout has variable behavior!

Reply:
Ah, thank you for reminding me about this utility; I've updated the tests accordingly.